relay-event-normalization/src/event.rs

```diff
@@ -578,18 +577,6 @@ fn normalize_dist(distribution: &mut Annotated<String>) {
     });
 }
 
-/// Defaults the `platform` and `level` required attributes.
-fn normalize_platform_and_level(event: &mut Event) {
```

Comment:
These are normalized in
https://github.com/getsentry/relay/blob/9bf4cbe9a5a2be8ac3d436a666724fd0586d1b17/relay-event-normalization/src/event.rs#L1030-L1033